packages/slider/test/benchmark/test-basic.ts

```diff
@@ -22,5 +22,6 @@ measureFixtureCreation(html`
         max="100"
         label="Opacity"
         id="opacity-slider"
+        .getAriaHandleText=${(value: number) => value.toString()}
```

Comment:
I appreciate leveraging this in a proof context, but we should gate as outlined above or test the raw perf here going forward.

Reply:
Removed